BT-Tracking/Services/RemoteValues.swift

```diff
@@ -99,17 +100,26 @@ struct RemoteValues {
 
         .emergencyNumber: 1212,
         
-        .helpMarkdown: helpMarkdownBackup
+        .helpMarkdown: helpMarkdownBackup,
+        .dataCollectionMarkdown: dataCollectionMarkdownBackup
     ]
 
-    private static var helpMarkdownBackup: String {
+    private static func localValue(forResource resource: String, withExtension extension: String, withKey key: String) -> String {
         guard
-            let path = Bundle.main.url(forResource: "HelpMarkdownBackup", withExtension: "strings", subdirectory: nil),
+            let path = Bundle.main.url(forResource: resource, withExtension: `extension`),
```

Comment:
Please use " instand of `

Reply:
then that would be treated as a String, wouldn't it? 🤔  @augard

Reply:
Then that would be treated as a String, wouldn't it? 🤔

Reply:
Then that would be treated as a String, wouldn't it? 🤔 @augard

Reply:
Ah, sorry, ignore my comment